tasks/migrations/0001_initial.py

```diff
@@ -27,8 +27,11 @@ class Migration(migrations.Migration):
                 ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('title', models.CharField(max_length=200)),
                 ('created_date', models.DateTimeField(auto_now_add=True)),
-                ('priority', enumchoicefield.fields.EnumChoiceField(default=tasks.models.Priority(1), enum_class=tasks.models.Priority, max_length=1)),
-                ('status', enumchoicefield.fields.EnumChoiceField(default=tasks.models.Status(1), enum_class=tasks.models.Status, max_length=1)),
+                ('priority', enumchoicefield.fields.EnumChoiceField(default=tasks.models.Priority(1),
```

Comment:
just be consistent, in 0002_initial.py you write the value and here you write the number.
I think it is better to write it with value: 
tasks.models.Priority.LOW